rust/parquet/src/util/bit_util.rs

```diff
@@ -124,11 +125,7 @@ where
 /// Returns the ceil of value/divisor
 #[inline]
 pub fn ceil(value: i64, divisor: i64) -> i64 {
-    let mut result = value / divisor;
-    if value % divisor != 0 {
-        result += 1
-    };
-    result
+    value / divisor + ((value % divisor != 0) as i64)
```

Comment:
Performance notes:

Small but measurable performance tweak, we can elide the branch and add on the `+1` from the gaurentees about the representation of the boolean